src/Native/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -26,6 +26,12 @@ if (OPENSSL_TLSV12)
     add_definitions(-DHAVE_TLS_V1_2=1)
 endif()
 
+include(CheckStructHasMember)
+check_struct_has_member ("SSL_CIPHER" algorithms openssl/ssl.h SSL_CIPHER_ALGORITHMS)
+if (SSL_CIPHER_ALGORITHMS)
+    add_definitions(-DHAVE_SSL_CIPHER_ALGORITHMS=1)
```

Comment:
Agreed. I can switch it, but was having trouble with the naming. Any suggestions?
- HAVE_SSL_CIPHER_ALGORITHM_ENC
- HAVE_SSL_CIPHER_NO_ALGORITHMS
- HAVE_SSL_CIPHER_SPLIT_ALGORITHMS